SmartDeviceLink/SDLProtocolListener.h

```diff
@@ -17,11 +17,6 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)handleProtocolEndServiceNAKMessage:(SDLProtocolMessage *)endServiceNAK;
 
 // Older protocol handlers
-- (void)handleProtocolStartSessionACK:(SDLServiceType)serviceType sessionID:(Byte)sessionID version:(Byte)version __deprecated_msg("use handleProtocolStartSessionACKMessage: instead");
-- (void)handleProtocolStartSessionACK:(SDLProtocolHeader *)header __deprecated_msg("use handleProtocolStartSessionACKMessage: instead");
-- (void)handleProtocolStartSessionNACK:(SDLServiceType)serviceType __deprecated_msg("use handleProtocolStartSessionNAKMessage: instead");
-- (void)handleProtocolEndSessionACK:(SDLServiceType)serviceType __deprecated_msg("use handleProtocolEndSessionACKMessage: instead");
-- (void)handleProtocolEndSessionNACK:(SDLServiceType)serviceType __deprecated_msg("use handleProtocolEndSessionNAKMessage: instead");
 - (void)handleHeartbeatForSession:(Byte)session;
```

Comment:
`handleProtocolStartSessionACK`, `handleProtocolStartSessionNACK`, `handleProtocolEndSessionACK`, and `handleProtocolEndSessionNACK` are all being used in the `SDLStreamingMediaLifecycleManager`. This means that video is not streaming because setup does not complete.